feat(changelog): transactional-email-added-activity-report-settings 2024-11-29 (#4064)
scaleway/docs-content: changelog/november2024/2024-11-29-transactional-email-added-activity-report-settings.mdx

---

> title: Activity Report Settings

Reviewer: ```suggestion
title: Activity report settings
```

status: added
author:
    fullname: 'Join the #transactional-email channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-11-29
category: managed-services
product: transactional-email
---

> You can now configure the activity report's delivery with a customizable monthly or weekly frequency. Additionally, you can enable or disable the report as needed either through the [console](https://console.scaleway.com/transactional-email/settings) or [via the API](https://www.scaleway.com/en/developers/api/transactional-email/#path-project-settings-update-project-settings).
> 

Reviewer: ```suggestion
You can now customize your activity report to a monthly or weekly frequency. Additionally, you can enable or disable the report as needed either through the [Scaleway console](https://console.scaleway.com/transactional-email/settings) or [the API](https://www.scaleway.com/en/developers/api/transactional-email/#path-project-settings-update-project-settings).
```
